vdp/pipeline/v1beta/pipeline.proto

```diff
@@ -1895,6 +1919,21 @@ message ListPipelineRunsResponse {
   int32 page_size = 4 [(google.api.field_behavior) = OUTPUT_ONLY];
 }
 
+// ListPipelineRunsByCreditOwnerResponse is the response message for ListPipelineRunsByCreditOwner.
+message ListPipelineRunsByCreditOwnerResponse {
+  // The list of pipeline runs.
+  repeated PipelineRun pipeline_runs = 1 [(google.api.field_behavior) = OUTPUT_ONLY];
+
+  // The total number of pipeline runs matching the request.
+  int64 total_size = 2 [(google.api.field_behavior) = OUTPUT_ONLY];
```

Comment:
Please use int32. Otherwise, it will become string by grpc-gateway.

Reply:
fixed